util/src/main/kotlin/ProcessCapture.kt

```diff
@@ -113,7 +114,7 @@ fun checkCommandVersion(
         ignoreActualVersion: Boolean = false,
         transform: (String) -> String = { it }
 ) {
-    val actualVersion = getCommandVersion(command, versionArguments, workingDir, transform)
+    val actualVersion = Semver(getCommandVersion(command, versionArguments, workingDir, transform), Semver.SemverType.LOOSE)
```

Comment:
Actually, we should fix this at the caller side by making the requirement a `LOOSE` instead of a `STRICT` / `NPM` version. We should be able to do this by changing `buildNPM` to `buildLoose` in this line:

https://github.com/heremaps/oss-review-toolkit/blob/12171c76cdc59f59c257e57c9b8e22908ed5d650/analyzer/src/main/kotlin/managers/NPM.kt#L116

Reply:
Changing Requirement.buildNPM to Requirement.buildLoose wouldn't fix this particular problem because that describes the requirement rather than what was detected. For the detection it should probably always be loose so you're resilient to tools that don't use strict semver in their --version output.